### ISSUE TITLE
Version 5.2 fixed the infinite loop of IOException but cannot stop the query now 

### ISSUE THREAD

**Issue by YuanchengJiang** [neo4j/neo4j#13002]
```
neo4j@neo4j> ONE QUERY COSTS TOO MUCH TIME;
^CStopping query...
^CStopping query...
^CStopping query...
^CStopping query...
^CStopping query...
^CStopping query...
^CStopping query...
^CStopping query...  <-- cannot stop here :(
```

**Comment by pontusmelke** [comment]
Thanks for reporting!

We are aware of this issue and we are working on providing a patch. Will update here when we have a fix.

Best regards,
Pontus

**Comment by loveleif** [comment]
Thanks again, the issue should be fixed in 5.7.0.
